data/json/npcs/talk_tags.json

```
@@ -242,7 +288,15 @@
       "you're a dead man",
       "you'll taste my <mywp>",
       "you're <swear> dead",
-      "<name_b>, <ill_kill_you>"
+      "<name_b>, <ill_kill_you>",
+      "Only one of us is leaving here alive.",
+      "Prepare thyself for punishment, foul knave!",
+      "Make peace with your gods.",
+      "If you worship any gods, now is a good time to start praying.",
+      "Your life is now forfeit",
+      "When I'm through with you, there won't be enough left to identify.",
```

```suggestion
      "when I'm through with you, there won't be enough left to identify",
```
This one sounds weird.